tests/test_client_functional.py

```diff
@@ -2032,6 +2033,30 @@ def dumps(obj):
     resp.close()
 
 
+async def test_set_cookies_max_age_overflow(aiohttp_client) -> None:
+
+    async def handler(request):
+        ret = web.Response()
+        ret.headers.add('Set-Cookie',
+                        'overflow=overflow; '
+                        'HttpOnly; Path=/'
+                        " Max-Age=" +
+                        str(int(datetime.datetime.max.timestamp())) + "; ")
```

Comment:
👍

Reply:
Unfortunately, the very act of asking for the time stamp can trigger an overflow exception, based on the current timezone! See the [appveyor test failures](https://ci.appveyor.com/project/aio-libs/aiohttp/builds/27616968/job/4i6eq555evqijl56#L3419):

```python
ERROR    aiohttp.server:web_protocol.py:380 Error handling request
Traceback (most recent call last):
  File "C:\projects\aiohttp\aiohttp\web_protocol.py", line 405, in _handle_request
    resp = await self._request_handler(request)
  File "C:\projects\aiohttp\aiohttp\web_app.py", line 330, in _handle
    resp = await handler(request)
  File "C:\projects\aiohttp\tests\test_client_functional.py", line 2044, in handler
    str(int(datetime.datetime.max.timestamp())) + "; ")
OverflowError: timestamp out of range
```

`datetime.max` Is a naive datetime value and has no timezone but a timestamp is always a value relative to 1970-01-01 midnight UTC, so Python uses the local timezone to interpret naive values when converting. In timezones west from UTC that requires adding hours to the datetime value, and so we get an overflow here.

Reply:
Oops. I guess I just need to choose a static large int value to avoid that issue.